src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -48,7 +48,7 @@ public CommentAttributes(String courseId, String giverEmail, CommentParticipantT
         this.giverEmail = giverEmail;
         this.recipientType = recipientType == null ? CommentParticipantType.PERSON : recipientType;
         this.recipients = recipients;
-        this.commentText = commentText;
+        this.commentText = commentText == null ? null : new Text(Sanitizer.sanitizeForRichText(commentText.getValue()));
```

Comment:
Just thinking, is it possible to make a `sanitizeForRichText` method that takes in `Text` parameter? It can be reused here and potentially many other places.